src/test/java/com/google/sps/NonPersistentUserRepositoryTest.java

```diff
@@ -0,0 +1,54 @@
+package com.google.sps;
+
+import com.google.sps.data.NonPersistentUserRepository;
+import com.google.sps.data.User;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** */
+@RunWith(JUnit4.class)
+public final class NonPersistentUserRepositoryTest {
```

Comment:
Maybe add some additional test for
1. getUsers() 
2. Adding the same user multiple times
3. some more complex sequence (add->remove->add, add -> add -> remove -> add)

Reply:
Thanks for the suggestion, I added all of these tests.